documentation/modules/managing/con-producer-config-properties.adoc

```diff
@@ -0,0 +1,203 @@
+// This module is included in the following files:
+//
+// assembly-client-config.adoc
+
+[id='con-producer-config-properties-{context}']
+= Kafka producer configuration tuning
+
+Use a minimum producer configuration with optional properties that are tailored to specific uses cases.
+
+Adjusting your configuration to maximize throughput might increase latency or vice versa.
+You will need to experiment and tune your producer configuration to get the balance you need.
+
+== Minimum producer configuration
+
+Connection and serializer properties are required for every producer.
+Generally, it is good practice to add a client id for tracking,
+and use compression on the producer to reduce batch sizes in requests.
+
+In this minimal configuration:
+
+* The order of messages in a partition is not guaranteed
```

Comment:
This is the opposite of the statement that is always mentioned when you talk about messages order in Kafka.
I guess you are saying that based on retries and inflight requests (so retries > 0 and inflight > 1 can drive to change the order) but I think that we should be clear about that otherwise we can confuse people reading since the beginning. Maybe we could say "The order of messages can be guaranteed at partition level only" or something like that @tombentley wdyt?

Reply:
I agree on the fact that the statement is wrong but I still think that "The order of messages can be guaranteed at partition level only" much better than saying "is not guaranteed" or at least we should add that it's not guaranteed if not configured in the proper way. I thought that my "can be" could open the possibility and the understanding that it can be guaranteed if configured to do so. 
Btw it's not just about our doc ... maybe every developer in the world having a Kafka presentation slides deck should change his slides at this point (me included).

Reply:
I think the section is clear that this is referring to the basic config. That is, before we look at the config that adds the guarantees

Reply:
Ok, so ... "In this basic configuration" ...should be the key. I really hope that other readers will pay more attention than me :-)